docs/README.md

```diff
@@ -0,0 +1,113 @@
+# Scylla documentation
+
+This is the repository for [Scylla documentation site](http://docs.scylladb.com/)
+
+To report an issue with the documentation, please use GitHub issues.
+
+This repository accepts GitHub pull requests.
+
+**Send pull requests to the master branch, not gh-pages.  gh-pages will be overwritten by deploy without warning.**
```

Comment:
What's gh-pages and how would anyone think to send pull requests there?

Reply:
Sphinx generate HTML  from the source, and the HTML is pushed to branch gh-pages
By default, Github pages publish the gh-pages branch.

Reply:
Ok, but why would anybody think they need to send PRs against a branch (gh-pages) that they don't even see?
It's not important to fix this now - you can reconsider it later when you rethink the README.md (which you said you are planning to anyway).